README.md

```diff
@@ -219,20 +218,20 @@ Licensed under the [GNU General Public License, Version 3](LICENSE).
    There is an environment var `CUDA_DEVICE_ORDER` which tells the Nvidia CUDA driver how to enumerates the graphic cards.
    Following values are valid:
 
-   * FASTEST_FIRST (Default) - causes CUDA to guess which device is fastest using a simple heuristic.
-   * PCI_BUS_ID - orders devices by PCI bus ID in ascending order.
+   * `FASTEST_FIRST` (Default) - causes CUDA to guess which device is fastest using a simple heuristic.
+   * `PCI_BUS_ID` - orders devices by PCI bus ID in ascending order.
 
-   To prevent some unwanted changes in the order of your CUDA devices you **might set the environment to `PCI_BUS_ID`**.
+   To prevent some unwanted changes in the order of your CUDA devices you **might set the environment variable to `PCI_BUS_ID`**.
    This can be done:
 
    * Linux:
-     * Adapt /etc/environment file and add a line `CUDA_DEVICE_ORDER=PCI_BUS_ID`
+     * Adapt the `/etc/environment` file and add a line `CUDA_DEVICE_ORDER=PCI_BUS_ID`
      * Adapt your start script launching ethminer and add a line `export CUDA_DEVICE_ORDER=PCI_BUS_ID`
 
    * Windows:
      * Adapt your environment using the control panel (just search `setting environment windows control panel` using your favorite search engine)
-     * Adapt your start (.bat) file launching ethminer and add a line `setx CUDA_DEVICE_ORDER=PCI_BUS_ID` or `set CUDA_DEVICE_ORDER=PCI_BUS_ID`
-     * For more details about `setx` and `set` see <https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/setx>
```

Comment:
I removed the `set` reference because this link does not show any info about `set`, just `setx`.

Reply:
In fact I should change this to use `set` because `setx` is permanent while `set` lives only for the script's cycle.